### ISSUE TITLE
Incorrect description of control panel in Scene Summary.

### ISSUE THREAD

**Issue by terracoda** [phetsims/resistance-in-a-wire#130]
In RC testing, @phet-steele pointedg out that RIAW does not have a mute sound button.

The reference to the mute sound button needs to be removed from the end of the opening paragraph.

Please change:
" In the Control Panel, buttons mute sound or reset the sim."

To:
"The Control Panel has a button to reset the sim."

**Comment by terracoda** [comment]
Self-assigning. I will make this small change.

**Comment by terracoda** [comment]
commit 0f4757b68c104dceefa343ffec355034064cfda0 fixes this issue.

**Comment by terracoda** [comment]
Waiting for response from @phet-steele re https://github.com/phetsims/QA/issues/92 before closing this issue.

**Comment by terracoda** [comment]
Thanks @phet-steele. I am closing this issue.
